themes/ddbasic/sass/components/ting-subsearch.scss

```diff
@@ -0,0 +1,130 @@
+// ting_subsearch styling.
+
+@import '../base.scss';
+
+.ting-subsearch-message-wrapper {
+  background-color: $grey-light;
+  padding-top: 30px;
+
+  .ting-subsearch-message {
+    max-width: 1124px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
```

Comment:
Can we do without the wrapper by using flexbox with a single centered element with a max-width instead?

Reply:
Will try. Maybe there will be some issues with background color.

Reply:
Removed wrapper.